x-pack/plugins/infra/public/components/asset_details/header/header.tsx

```diff
@@ -43,50 +58,66 @@ export const Header = ({ nodeType, node, tabs, links, compact, onTabsStateChange
     showTab(tabId);
   };
 
-  const tabEntries = tabs.map((tab) => (
-    <EuiTab
-      {...tab}
-      key={tab.id}
-      onClick={() => onTabClick(tab.id)}
-      isSelected={tab.id === activeTabId}
-    >
-      {tab.name}
-    </EuiTab>
-  ));
+  const tabLinkComponents = {
+    [FlyoutTabIds.LINK_TO_APM]: (tab: Tab) => (
+      <TabToApmTraces nodeName={node.name} apmField={APM_FIELD} {...tab} />
+    ),
+    [FlyoutTabIds.LINK_TO_UPTIME]: (tab: Tab) => (
+      <TabToUptime nodeName={node.name} nodeType={nodeType} nodeIp={node.ip} {...tab} />
+    ),
+  };
 
-  const linkComponent = {
-    apmServices: <LinkToApmServices hostName={node.name} apmField={'host.hostname'} />,
-    uptime: <LinkToUptime nodeType={nodeType} node={node} />,
+  const topCornerLinkComponents: Record<LinkOptions, JSX.Element> = {
+    nodeDetails: (
+      <LinkToNodeDetails nodeId={node.id} nodeType={nodeType} currentTime={currentTimeRange.to} />
+    ),
+    alertRule: <LinkToAlertsRule onClick={overrides?.alertRule?.onCreateRuleClick} />,
+    apmServices: <LinkToApmServices nodeName={node.name} apmField={APM_FIELD} />,
+    uptime: <LinkToUptime nodeName={node.name} nodeType={nodeType} nodeIp={node.ip} />,
   };
 
-  const headerLinks = links?.map((link, index) => (
-    <EuiFlexItem key={index} grow={false}>
-      {linkComponent[link]}
-    </EuiFlexItem>
-  ));
+  const tabEntries = tabs.map(({ name, ...tab }) => {
+    if (Object.keys(tabLinkComponents).includes(tab.id)) {
+      return (
+        <React.Fragment key={tab.id}>
+          {tabLinkComponents[tab.id as keyof typeof tabLinkComponents]({ name, ...tab })}
```

Comment:
I'm not sure if I understood your question. But on Inventory UI APM and Uptime links are shown as tabs

<img width="1486" alt="image" src="https://github.com/elastic/kibana/assets/2767137/f6ed33f0-9536-4a9a-9071-7c9530131aac">

As for the links at the top. Similar to the tabs, they are configurable. The client decides what to show.
On the Inventory UI, it would look like this:

<img width="1486" alt="image" src="https://github.com/elastic/kibana/assets/2767137/036a2620-5cc4-42b2-aa98-b7d32685b0af">